openpype/modules/deadline/abstract_submit_deadline.py

```diff
@@ -460,7 +460,22 @@ def process(self, instance):
         self.plugin_info = self.get_plugin_info()
         self.aux_files = self.get_aux_files()
 
-        self.process_submission()
+        job_id = self.process_submission()
```

Comment:
@antirotor could you please have a quick look at this. I think this should be good to go and get merged.